.github/workflows/release.yml

```
@@ -0,0 +1,58 @@
+name: tagged-release
+
+on:
+  push:
+    tags:
+      - 'v[1-9].[0-9]+.[0-9]+'
+
+jobs:
+  github-release:
+    name: 'GitHub Release'
+    runs-on: 'ubuntu-latest'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          # https://github.com/marketplace/actions/goreleaser-action#usage
+          # note the fetch-depth: 0 input in Checkout step. It is required for
+          # the changelog to work correctly
+          fetch-depth: 0
+      - name: Install Go
+        uses: actions/setup-go@v2
+      - name: Restore cache
+        uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Run GoReleaser
```

```suggestion
      - name: Login to Docker Hub
        uses: docker/login-action@v1
        with:
          username: sammytheshark
          password: ${{ secrets.DOCKER_PASSWORD }}

      - name: Run GoReleaser
```

Some time before the goreleaser step, we need to log into Docker Hub